app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,133 @@
+  classDef serviceCatalog fill:#e1f7d5,stroke:#333,stroke-width:2px;
+  classDef billingInventory fill:#f0f0f0,stroke:#333,stroke-width:2px;
+
+  %% Define the resources
+  GitHub[<img src="https://img.icons8.com/ios/50/000000/github.png" width="24" height="24"/> GitHub]:::resource
+  PagerDuty[<img src="https://play-lh.googleusercontent.com/E-zhAf4KJ6JDDXmQfQxBprn2sATGYUMkOEqLQX5HAQQtiwDZJg4c8sQd7deb6nCZCwU=w480-h960-rw" width="24" height="24"/> PagerDuty]:::resource
+  GatewayManager[<img src="https://raw.githubusercontent.com/Kong/docs.konghq.com/main/app/assets/images/icons/kong-gradient.svg" width="15" height="15"/> Gateway Manager]:::resource
+
+  %% Define the Konnect Service Catalog and its subgraphs
+  subgraph KonnectServiceCatalog[Konnect Service Catalog]
+    direction TB
+
+    %% Define the Billing Service subgraph inside Konnect Service Catalog
+    subgraph BillingService[Billing Service]
+      direction LR
+      KongGatewayService[Gateway service 1]
+      PagerDutyServiceMapping[PagerDuty service 1]
+      GitHubRepoMapping[GitHub repository 1]
+    end
+
+    %% Define the Inventory Service Catalog Service subgraph inside Konnect Service Catalog
+    subgraph InventoryService[Inventory Service]
+      direction LR
+      SomeOtherService[Gateway service 2]
+      AnotherService[PagerDuty service 2]
+      DifferentRepo[GitHub repository 2]
+    end
+
+  end
+
+  %% Connect the resources to the Konnect Service Catalog
+  GitHub -- "GitHub repositories" --> KonnectServiceCatalog
+  PagerDuty -- "PagerDuty services" --> KonnectServiceCatalog
+  GatewayManager -- "Gateway services" --> KonnectServiceCatalog
+
+  %% Style the subgraphs
+  class KonnectServiceCatalog konnect
+  class BillingService serviceCatalog
+  class InventoryService serviceCatalog
+{% endmermaid %}
+<!-- vale on-->
+
+> Figure 1: This diagram shows how you can use both external integrations, like GitHub and PagerDuty, as well as built-in integrations like Gateway Manager to pull resources into Service Catalog. You can then map those resources (like GitHub repositories, PagerDuty services, and Gateway services) to Service Catalog services.
+
+## Service Catalog use cases
+
+| You want to... | Then use... |
+| -------------- | ----------- |
+| Keep track of your organization's resource ownership by mapping teams to Service Catalog services. | Add the Service Catalog service's owner when you [create a new Service Catalog service in {{site.konnect_short_name}}](https://cloud.konghq.com/service-catalog/create-service) |
+| Gain visibility into all your Services, including unrecognized or undiscovered APIs in your organization.  | [Service Catalog integrations](https://cloud.konghq.com/service-catalog/integrations) |
+| Consolidate key Service Catalog service health metrics, documentation, and API specs into a single dashboard, allowing you to interact with other tools from one place. | [Service Catalog dashboard](https://cloud.konghq.com/service-catalog/) |
```

Comment:
I could go either way, but I would probably call this a "registry" or a "list", rather than a "dashboard".